assets/queries/cloudFormation/user_data_base64_contains_encoded_private_key/test/negative.yaml

```diff
@@ -0,0 +1,13 @@
+---
+Resources:
+  myLaunchConfig:
```

Comment:
Change "myLaunchConfig" to "myLaunchConfig2"

Reply:
done

Reply:
Why rename for different samples?

Reply:
done